style.css

```diff
@@ -0,0 +1,628 @@
+    line-height: 20px;
+    padding-bottom: 5px;
+    font-weight: normal;
+}
+
+.about-me__work{
+    font-family: "Maven", Arial, sans-serif;
+    font-size: 18px;
+    word-spacing: -1px;
+    padding-bottom: 14px;
+    color: #20816d;
+    font-weight: normal;
+}
+
+.about-me__content p{
+    font-family: "Lato", Arial, sans-serif; 
+    text-align: justify;
+    line-height: 22px;
+    font-size: 12px;    
+}
+
+.about-me__social{
+    position: absolute;
+    bottom: 0;
+    right: 1px;
+    font-size: 0;
+}
+
+.about-me__social-item{
+    display: inline-block;
+    font-size: 13px;
+    line-height: 24px;
+    text-align: center;
+    width: 25px;
+    height: 24px;
+    background-color: #abddd7;
+    cursor: pointer;
+    -moz-transition: background-color 0.5s ease-in-out;
+    -webkit-transition: background-color 0.5s ease-in-out;
+    transition: background-color 0.5s ease-in-out;
+}
+
+.about-me__social-item a{
+    color:#2eb398;
+}
+
+.about-me__social-item:nth-of-type(2){
+    background-color: #6ccbbe;
+}
+
+.about-me__social-item:nth-of-type(3),
+.about-me__social-item:nth-of-type(6){
+    background-color: #7fd0c5;
+}
+
+.about-me__social-item:hover{
+    background-color: #FFFFFF;
+}
+
+/* icons */
+
+.about-me__social-item i:before,
+.contacts__item i:before{
+    font-family: "fontello";
+    font-style: normal;
+    font-weight: normal;
+    speak: none;
+    display: inline-block;
+    text-decoration: inherit;
+    width: 1em;
+    margin-right: .2em;
+    text-align: center;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1em;
+    margin-left: .2em;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+
+.about-me__social-item.gplus i:before { content: '\e800'; } /* '' */
+.about-me__social-item.twitter i:before { content: '\f099'; } /* '' */
+.about-me__social-item.facebook i:before { content: '\f09a'; } /* '' */
+.about-me__social-item.linkedin i:before { content: '\f0e1'; } /* '' */
+.about-me__social-item.dribbble i:before { content: '\f17d'; } /* '' */
+.about-me__social-item.pinterest i:before { content: '\f231'; } /* '' */
+
+/* get-in-touch */
+
+.get-in-touch{
+    background-color: #323232;
+}
+
+.get-in-touch__container{
+    width: 1020px;
+    margin: 0 auto;
+    padding: 13px 0;
+}
+
+.get-in-touch__wrap{
+    margin: 0 30px;
+    padding: 45px 90px 52px;
+    border-top: 1px #2b2b2b solid;
+    border-bottom: 1px #2b2b2b solid;
+    font-family: "Maven", Arial, sans-serif;
+    color:#bebebe;
+}
+
+.get-in-touch__section-name{
+    font-size: 30px;
+    font-weight: normal;
+    color: #e0e0e0;
+}
+
+.get-in-touch__title{
+    font-size: 18px;
+    padding: 0px 1px 21px;
+    line-height: 19px;
+}
+
+.get-in-touch__feedback-form{
+    width: 580px;
+    font-family: "Lato", Arial, sans-serif;
+    display: inline-block;
+}
+
+.feedback-form__name,
+.feedback-form__email,
+.feedback-form__subject,
+.feedback-form__message{
+    width: 100%;
```

Comment:
it will be proper, if you use flex.
Set for form 
    display: flex;
    flex-direction: column;
and for button 
   align-self: flex-end;
As a result you don't use float and don't use width: 100%;